### ISSUE TITLE
Lazy load view solutions page

### ISSUE THREAD

**Issue by pradeepbishnoi** [oxalorg/4ever-clojure#34]
Currently if you open the 'view solutions' of problem 1 - it goes into a frozen state and then after a minute or so it shows up all solutions. Will be good user experience to load first 5-10 solutions and then on scroll down bring the next 10 etc.

https://4clojure.oxal.org/#/problem/1/solutions 

Also, can think for creating a Set and remove duplicate solutions. Look at the screenshot with duplicate solution for problem 1 (it might be the case for other as well). Better will be to clean the solutions data one time (remove duplicate) and store it.

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/1634146/125307845-096c9f80-e34e-11eb-87e0-c319517d5a60.png">

**Comment by borkdude** [comment]
Yes. It might not be so much the amount of data, but the amount of rendering might take the most of the time. React is known to not work well with large lists. So maybe it's good to first find out if that's the bottleneck or not. React Virtualized offers a solution for rendering larger lists well, perhaps there are also other solutions. @oxalorg
